### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by rudiger-empira** [pgrimaud/instagram-user-feed#193] [Q&A] [answered]
This package is great, thank you

**Comment by pgrimaud** [comment, marked as answer]
🚀

**Reply by rendyam** [reply]
Hi,

I want to asking about how come this repo get the bunch of data? (such as getProfile() method)

It took whole of feeds from a profile. Is this using a legal method? (According to Instagram Dev ToS or Privacy Policy)

Thank you
